packages/timeline-state-resolver/src/integrations/vmix/index.ts

```diff
@@ -1015,6 +1056,7 @@ export interface VMixStateExtended {
 		Fullscreen2: VMixOutput
 	}
 	inputLayers: { [key: string]: string }
+	desiredScripts: string[]
```

Comment:
Why is this property called `desiredScripts`? Isn't the entirety of the state - _the desired state_? I think it might create some confusion down the line. I'd call it `scripts` or `runningScripts`, since as far as TSR is concerned, these scripts are running and, for example, it won't start the script again, if it thinks it is running because one timelineObject is replaced by another, with the same `SCRIPT`.